fix(deps): replace `react-native-aes-crypto-forked` with `react-native-aes-crypto`

legobeat opened the pull request:
## **Description**

Instead of pulling in `react-native-aes-crypto-forked` from a git-commit and then patching it (!(?)), this migrates to a published version of the package under the name `@metamask/react-native-aes-crypto` with the same effect.

The separate dependency `react-native-aes-crypto` is left unaddressed.

## **Related issues**
- Origin: #1755
- #1795
- #9387 
- #5324 
### Blocked by
- [ ] https://github.com/MetaMask/react-native-aes-crypto-forked/pull/6

 
## **Manual testing steps**


## **Screenshots/Recordings**

### **Before**

### **After**

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

rickycodes committed:
deps: react-native-aes-crypto-forked -> @metamask/react-native-aes-crypto

legobeat committed:
fix: rename react-native-aes-crypto-forked patch

legobeat committed:
fix: update react-native config with @metamask/react-native-crypto dep

legobeat committed:
ios: update react-native module paths in xcodeproj

legobeat committed:
chore: update depcheckrc

legobeat committed:
deps: @metamask/react-native-crypto-aes@1.2.1->^1.2.2

legobeat commented:
iOS build error needs resolving:

```
WriteAuxiliaryFile /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.LinkFileList (in target 'RCTAesForked' from project 'RCTAesForked')
    cd /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios
    write-file /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.LinkFileList
WriteAuxiliaryFile /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.LinkFileList (in target 'RCTAesForked' from project 'RCTAesForked')
    cd /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios
    write-file /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.LinkFileList
CompileC /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.o /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RCTAesForked' from project 'RCTAesForked')
    cd /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios
    clang -x objective-c -target x86_64-apple-ios9.0-simulator -fmessage-length\=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fno-color-diagnostics -std\=gnu99 -fobjc-arc -fmodules -fmodules-cache-path\=/Users/vagrant/git/ios/build/ModuleCache.noindex -fmodules-prune-interval\=86400 -fmodules-prune-after\=345600 -fbuild-session-file\=/Users/vagrant/git/ios/build/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror\=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror\=return-type -Wunreachable-code -Wno-implicit-atomic-properties -Werror\=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror\=objc-root-class -Wno-arc-repeated-use-of-weak -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wno-non-literal-null-conversion -Wno-objc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wno-deprecated-implementations -Wno-implicit-fallthrough -DDEBUG\=1 -DOBJC_OLD_DISPATCH_PROTOTYPES\=0 -isysroot /Applications/Xcode-15.0.1.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wno-comma -Wno-block-capture-autoreleasing -Wno-strict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version\=2 -fobjc-legacy-dispatch -index-store-path /Users/vagrant/git/ios/build/Index.noindex/DataStore -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-generated-files.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-own-target-headers.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-all-non-framework-target-headers.hmap -ivfsoverlay /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/all-product-headers.yaml -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-project-headers.hmap -I/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator/include -I/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -I../../../react-native/React -I../../../react-native/React/AccessibilityResources -I../../../react-native/React/AppSetup -I../../../react-native/React/Base -I../../../react-native/React/CoreModules -I../../../react-native/React/CxxBridge -I../../../react-native/React/CxxLogUtils -I../../../react-native/React/CxxModule -I../../../react-native/React/CxxUtils -I../../../react-native/React/DevSupport -I../../../react-native/React/FBReactNativeSpec -I../../../react-native/React/Fabric -I../../../react-native/React/Inspector -I../../../react-native/React/Modules -I../../../react-native/React/Profiler -I../../../react-native/React/Tests -I../../../react-native/React/UIUtils -I../../../react-native/React/Views -I../../../react-native/React/Base/Surface -I../../../react-native/React/Fabric/Mounting -I../../../react-native/React/Fabric/Surface -I../../../react-native/React/Fabric/Utils -I../../../react-native/React/Tests/Mounting -I../../../react-native/React/Tests/Text -I../../../react-native/React/Views/RefreshControl -I../../../react-native/React/Views/SafeAreaView -I../../../react-native/React/Views/ScrollView -I../../../react-native/React/Base/Surface/SurfaceHostingView -I../../../react-native/React/Fabric/Mounting/ComponentViews -I../../../react-native/React/Fabric/Mounting/ComponentViews/ActivityIndicator -I../../../react-native/React/Fabric/Mounting/ComponentViews/Image -I../../../react-native/React/Fabric/Mounting/ComponentViews/InputAccessory -I../../../react-native/React/Fabric/Mounting/ComponentViews/LegacyViewManagerInterop -I../../../react-native/React/Fabric/Mounting/ComponentViews/Modal -I../../../react-native/React/Fabric/Mounting/ComponentViews/Root -I../../../react-native/React/Fabric/Mounting/ComponentViews/SafeAreaView -I../../../react-native/React/Fabric/Mounting/ComponentViews/ScrollView -I../../../react-native/React/Fabric/Mounting/ComponentViews/Slider -I../../../react-native/React/Fabric/Mounting/ComponentViews/Switch -I../../../react-native/React/Fabric/Mounting/ComponentViews/Text -I../../../react-native/React/Fabric/Mounting/ComponentViews/TextInput -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedComponent -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedView -I../../../react-native/React/Fabric/Mounting/ComponentViews/View -I/Users/vagrant/node_modules/react-native/React -I/Users/vagrant/git/node_modules/ios/Pods/Headers/Public/React-Core -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources-normal/x86_64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources/x86_64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources -F/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator -MMD -MT dependencies -MF /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.d --serialize-diagnostics /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.dia -c /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m -o /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/RCTAesForked.o
In file included from /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m:10:
/Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.h:10:9: fatal error: 'React/RCTBridgeModule.h' file not found
#import <React/RCTBridgeModule.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~
CompileC /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/AesCryptForked.o /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/lib/AesCryptForked.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RCTAesForked' from project 'RCTAesForked')
    cd /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios
    clang -x objective-c -target x86_64-apple-ios9.0-simulator -fmessage-length\=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fno-color-diagnostics -std\=gnu99 -fobjc-arc -fmodules -fmodules-cache-path\=/Users/vagrant/git/ios/build/ModuleCache.noindex -fmodules-prune-interval\=86400 -fmodules-prune-after\=345600 -fbuild-session-file\=/Users/vagrant/git/ios/build/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror\=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror\=return-type -Wunreachable-code -Wno-implicit-atomic-properties -Werror\=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror\=objc-root-class -Wno-arc-repeated-use-of-weak -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wno-non-literal-null-conversion -Wno-objc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wno-deprecated-implementations -Wno-implicit-fallthrough -DDEBUG\=1 -DOBJC_OLD_DISPATCH_PROTOTYPES\=0 -isysroot /Applications/Xcode-15.0.1.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wno-comma -Wno-block-capture-autoreleasing -Wno-strict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version\=2 -fobjc-legacy-dispatch -index-store-path /Users/vagrant/git/ios/build/Index.noindex/DataStore -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-generated-files.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-own-target-headers.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-all-non-framework-target-headers.hmap -ivfsoverlay /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/all-product-headers.yaml -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-project-headers.hmap -I/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator/include -I/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -I../../../react-native/React -I../../../react-native/React/AccessibilityResources -I../../../react-native/React/AppSetup -I../../../react-native/React/Base -I../../../react-native/React/CoreModules -I../../../react-native/React/CxxBridge -I../../../react-native/React/CxxLogUtils -I../../../react-native/React/CxxModule -I../../../react-native/React/CxxUtils -I../../../react-native/React/DevSupport -I../../../react-native/React/FBReactNativeSpec -I../../../react-native/React/Fabric -I../../../react-native/React/Inspector -I../../../react-native/React/Modules -I../../../react-native/React/Profiler -I../../../react-native/React/Tests -I../../../react-native/React/UIUtils -I../../../react-native/React/Views -I../../../react-native/React/Base/Surface -I../../../react-native/React/Fabric/Mounting -I../../../react-native/React/Fabric/Surface -I../../../react-native/React/Fabric/Utils -I../../../react-native/React/Tests/Mounting -I../../../react-native/React/Tests/Text -I../../../react-native/React/Views/RefreshControl -I../../../react-native/React/Views/SafeAreaView -I../../../react-native/React/Views/ScrollView -I../../../react-native/React/Base/Surface/SurfaceHostingView -I../../../react-native/React/Fabric/Mounting/ComponentViews -I../../../react-native/React/Fabric/Mounting/ComponentViews/ActivityIndicator -I../../../react-native/React/Fabric/Mounting/ComponentViews/Image -I../../../react-native/React/Fabric/Mounting/ComponentViews/InputAccessory -I../../../react-native/React/Fabric/Mounting/ComponentViews/LegacyViewManagerInterop -I../../../react-native/React/Fabric/Mounting/ComponentViews/Modal -I../../../react-native/React/Fabric/Mounting/ComponentViews/Root -I../../../react-native/React/Fabric/Mounting/ComponentViews/SafeAreaView -I../../../react-native/React/Fabric/Mounting/ComponentViews/ScrollView -I../../../react-native/React/Fabric/Mounting/ComponentViews/Slider -I../../../react-native/React/Fabric/Mounting/ComponentViews/Switch -I../../../react-native/React/Fabric/Mounting/ComponentViews/Text -I../../../react-native/React/Fabric/Mounting/ComponentViews/TextInput -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedComponent -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedView -I../../../react-native/React/Fabric/Mounting/ComponentViews/View -I/Users/vagrant/node_modules/react-native/React -I/Users/vagrant/git/node_modules/ios/Pods/Headers/Public/React-Core -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources-normal/x86_64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources/x86_64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources -F/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator -MMD -MT dependencies -MF /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/AesCryptForked.d --serialize-diagnostics /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/AesCryptForked.dia -c /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/lib/AesCryptForked.m -o /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/x86_64/AesCryptForked.o
CompileC /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.o /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m normal arm64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'RCTAesForked' from project 'RCTAesForked')
    cd /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios
    clang -x objective-c -target arm64-apple-ios9.0-simulator -fmessage-length\=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fno-color-diagnostics -std\=gnu99 -fobjc-arc -fmodules -fmodules-cache-path\=/Users/vagrant/git/ios/build/ModuleCache.noindex -fmodules-prune-interval\=86400 -fmodules-prune-after\=345600 -fbuild-session-file\=/Users/vagrant/git/ios/build/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror\=non-modular-include-in-framework-module -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror\=return-type -Wunreachable-code -Wno-implicit-atomic-properties -Werror\=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror\=objc-root-class -Wno-arc-repeated-use-of-weak -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wno-non-literal-null-conversion -Wno-objc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wno-deprecated-implementations -Wno-implicit-fallthrough -DDEBUG\=1 -DOBJC_OLD_DISPATCH_PROTOTYPES\=0 -isysroot /Applications/Xcode-15.0.1.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator17.0.sdk -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wno-comma -Wno-block-capture-autoreleasing -Wno-strict-prototypes -Wno-semicolon-before-method-body -fobjc-abi-version\=2 -fobjc-legacy-dispatch -index-store-path /Users/vagrant/git/ios/build/Index.noindex/DataStore -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-generated-files.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-own-target-headers.hmap -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-all-non-framework-target-headers.hmap -ivfsoverlay /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/all-product-headers.yaml -iquote /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/RCTAesForked-project-headers.hmap -I/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator/include -I/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include -I../../../react-native/React -I../../../react-native/React/AccessibilityResources -I../../../react-native/React/AppSetup -I../../../react-native/React/Base -I../../../react-native/React/CoreModules -I../../../react-native/React/CxxBridge -I../../../react-native/React/CxxLogUtils -I../../../react-native/React/CxxModule -I../../../react-native/React/CxxUtils -I../../../react-native/React/DevSupport -I../../../react-native/React/FBReactNativeSpec -I../../../react-native/React/Fabric -I../../../react-native/React/Inspector -I../../../react-native/React/Modules -I../../../react-native/React/Profiler -I../../../react-native/React/Tests -I../../../react-native/React/UIUtils -I../../../react-native/React/Views -I../../../react-native/React/Base/Surface -I../../../react-native/React/Fabric/Mounting -I../../../react-native/React/Fabric/Surface -I../../../react-native/React/Fabric/Utils -I../../../react-native/React/Tests/Mounting -I../../../react-native/React/Tests/Text -I../../../react-native/React/Views/RefreshControl -I../../../react-native/React/Views/SafeAreaView -I../../../react-native/React/Views/ScrollView -I../../../react-native/React/Base/Surface/SurfaceHostingView -I../../../react-native/React/Fabric/Mounting/ComponentViews -I../../../react-native/React/Fabric/Mounting/ComponentViews/ActivityIndicator -I../../../react-native/React/Fabric/Mounting/ComponentViews/Image -I../../../react-native/React/Fabric/Mounting/ComponentViews/InputAccessory -I../../../react-native/React/Fabric/Mounting/ComponentViews/LegacyViewManagerInterop -I../../../react-native/React/Fabric/Mounting/ComponentViews/Modal -I../../../react-native/React/Fabric/Mounting/ComponentViews/Root -I../../../react-native/React/Fabric/Mounting/ComponentViews/SafeAreaView -I../../../react-native/React/Fabric/Mounting/ComponentViews/ScrollView -I../../../react-native/React/Fabric/Mounting/ComponentViews/Slider -I../../../react-native/React/Fabric/Mounting/ComponentViews/Switch -I../../../react-native/React/Fabric/Mounting/ComponentViews/Text -I../../../react-native/React/Fabric/Mounting/ComponentViews/TextInput -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedComponent -I../../../react-native/React/Fabric/Mounting/ComponentViews/UnimplementedView -I../../../react-native/React/Fabric/Mounting/ComponentViews/View -I/Users/vagrant/node_modules/react-native/React -I/Users/vagrant/git/node_modules/ios/Pods/Headers/Public/React-Core -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources-normal/arm64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources/arm64 -I/Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/DerivedSources -F/Users/vagrant/git/ios/build/Build/Products/Debug-iphonesimulator -MMD -MT dependencies -MF /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.d --serialize-diagnostics /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.dia -c /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m -o /Users/vagrant/git/ios/build/Build/Intermediates.noindex/RCTAesForked.build/Debug-iphonesimulator/RCTAesForked.build/Objects-normal/arm64/RCTAesForked.o
In file included from /Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.m:10:
/Users/vagrant/git/node_modules/@metamask/react-native-aes-crypto/ios/RCTAesForked/RCTAesForked.h:10:9: fatal error: 'React/RCTBridgeModule.h' file not found
#import <React/RCTBridgeModule.h>
        ^~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

leotm committed:
deps: strip @metamask/react-native-crypto-aes

- package.json
- yarn.lock (run deduplicate)
- react-native.config.js (removed in RN 0.74)
- depcheck

leotm committed:
deps: strip @metamask/react-native-crypto-aes (pbxproj)

leotm commented:
> Removing it instead, now we get iOS build error in Bitrise:
> 
> ```
> 2024-06-13 19:28:34.316 xcodebuild[85737:403517] [MT] IDEFileReferenceDebug: [Load] <IDEFileReference, 0x600002828900: name:MetaMask path:group:MetaMask.xcodeproj> Failed to load container at path: /Users/vagrant/git/ios/MetaMask.xcodeproj, Error: Error Domain=Xcode3ProjectErrorDomain Code=2 "Xcode has encountered an error reading an invalid project file. The project file `/Users/vagrant/git/ios/MetaMask.xcodeproj` contains a PBXContainerItemProxy `650F2B9B24DC5FEC00C3B9C4` whose containerPortal does not exist." UserInfo={NSLocalizedDescription=Xcode has encountered an error reading an invalid project file. The project file `/Users/vagrant/git/ios/MetaMask.xcodeproj` contains a PBXContainerItemProxy `650F2B9B24DC5FEC00C3B9C4` whose containerPortal does not exist.}
> --- xcodebuild: WARNING: Unable to open project file '/Users/vagrant/git/ios/MetaMask.xcodeproj' in workspace 'MetaMask.xcworkspace'.
> 2024-06-13 19:28:51.557 xcodebuild[85737:403517] Writing error result bundle to /var/folders/yy/6kcn9mkd5svdbqnznwf474f00000gn/T/ResultBundle_2024-13-06_19-28-0051.xcresult
> xcodebuild: error: The workspace named "MetaMask" does not contain a scheme named "MetaMask". The "-list" option can be used to find the names of the schemes in the workspace.
> error Command failed with exit code 65.
> info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
> ```

yep `Xcode has encountered an error reading an invalid project file`, locally same

<img width="286" alt="Screenshot 2024-06-19 at 6 05 25 PM" src="https://github.com/MetaMask/metamask-mobile/assets/1881059/a5bc50ac-abc0-4907-a1a4-38e356ae753f">

suspecting it was https://github.com/MetaMask/metamask-mobile/pull/9540/commits/22bf1b09b64806cfd80bd4028df748a0c7ba9b36 going to revert

leotm committed:
Revert "deps: strip @metamask/react-native-crypto-aes (pbxproj)"

This reverts commit 22bf1b09b64806cfd80bd4028df748a0c7ba9b36.

leotm committed:
Revert "deps: strip @metamask/react-native-crypto-aes"

This reverts commit b700a298d33848d1a2e2c443f771a5ac0eef56db.

leotm commented:
@SocketSecurity ignore npm/@metamask/react-native-aes-crypto@1.2.2

prev author: gudahtt, new lgbot ok

leotm committed:
Fix iOS @metamask/react-native-aes-crypto build error

- Fix CompileC RCTAesForked.m failure
  - Fix fatal error: 'React/RCTBridgeModule.h' file not found in
    - #import <React/RCTBridgeModule.h>
- Remove libRCTAesForked.a via Xcode

leotm commented:
<img width="1053" alt="Screenshot 2024-06-19 at 8 55 13 PM" src="https://github.com/MetaMask/metamask-mobile/assets/1881059/48cdd96c-16a8-4279-9f28-50653633fe8d">

<img width="939" alt="Screenshot 2024-06-19 at 9 02 55 PM" src="https://github.com/MetaMask/metamask-mobile/assets/1881059/52a4260a-476c-4a04-a8df-f7ae9c160d50">

fixed in: https://github.com/MetaMask/metamask-mobile/pull/9540/commits/abb3a001669cedaf6cc9fa65a17df28ff069efda

legobeat committed:
Merge branch 'main' into deps-mm-react-native-aes-crypto

legobeat committed:
Merge branch 'main' into deps-mm-react-native-aes-crypto

legobeat committed:
Merge branch 'main' into deps-mm-react-native-aes-crypto

legobeat committed:
Merge branch 'main' into deps-mm-react-native-aes-crypto
